tests/unit/local/docker/test_lambda_container.py

```diff
@@ -511,7 +511,8 @@ def test_must_provide_container_env_vars(self, runtime):
 
         self.assertIsNotNone(container_env_vars)
 
-    @parameterized.expand([param(r) for r in set(RUNTIMES_WITH_BOOTSTRAP_ENTRYPOINT)])
+    # sort the parameters is essential making unit test names deterministic, which is required by pytest-xdist
+    @parameterized.expand([param(r) for r in sorted(set(RUNTIMES_WITH_BOOTSTRAP_ENTRYPOINT))])
```

Comment:
@mndeveci this is the fix I mentioned.

`pytest-xdist` needs to collect tests in each worker and raise an exception if the collection is not deterministic. Here we sort to ensure each worker can collect the same collection of tests.

Reply:
Do we need this kind of sorts anywhere in the codebase? Like the ones in `integration/deployment`?

Reply:
only where we have `set(...)`, list is fine as long as it has an order